docs/src/devdocs/FEValues.md

```diff
@@ -18,6 +19,13 @@ Ferrite.FunctionValues
 Ferrite.BCValues
 ```
 
+## Internal utilities
+```@docs
+Ferrite.embedded_det
```

Comment:
Missing? 
https://ferrite-fem.github.io/Ferrite.jl/previews/PR964/devdocs/FEValues/#Internal-utilities

Reply:
Have we renamed this function? :D